### PR TITLE
VBID-4245 Cortex Titan Metro Transit app updates: part 1

### PR DESCRIPTION
**Pull request by pbogunovich**
* Added support for dynamically programmed schedules using 'titan.schedule'
  parameter.
  * Falls back to default schedule in the case that the parameter is not set.
  * Added tests for dynamic schedules

### CODE REVIEW

**Comment by jawsthegame** [comment]
:+1:

File: app/main.coffee [hunk 1]
```diff
@@ -29,38 +29,54 @@ class Main
...
-      feedCachePeriod:  30 * 60 * 1000
+  _scheduleKeys: =>
+    if @_cortex.getConfig()['titan.schedule']?
+      console.log "Using titan.schedule:
+        #{@_cortex.getConfig()['titan.schedule']}"
+      @_cortex.getConfig()['titan.schedule']
+        .split(',').map (s) -> s.trim().toLowerCase()
+    else
+      console.log 'No titan.schedule parameter found. Using default schedule'
+      ['a','a','a','t','a','a','a','e','a','a','a','t']
```
Comment (chrismcbride) [inline app/main.coffee]: would it be cleaner to use some kind of constants for this? Like [AD, TRANSIT, ..] ?

**Comment by chrismcbride** [comment]
+2

File: manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "cortex-metro-transit",
-    "version": "0.0.7",
+    "version": "1.0.2",
```
Comment (natlownes) [inline manifest.json]: I don't *think* you have to bump these, I think `gulp pack` will make these match the version in package.json

File: app/main.coffee (hunk 1 above)
Comment (pbogunovich) [inline app/main.coffee]: Done

**Pull request merged**